### DISCUSSION TITLE
v1.0 RC – Can it outshine Next.js?

### DISCUSSION THREAD

**Discussion by tipiirai** [nuejs/nue#400] [Announcements]
This major update introduces a range of improvements while simplifying the system. The documentation has been greatly enhanced, and there's a completely new Markdown parser from the ground up, specifically designed to bring Nue’s content-first vision to life. Check out details:

https://nuejs.org/blog/nue-release-candidate/

<a href="https://nuejs.org/blog/nue-release-candidate/">
<img src="https://nuejs.org/img/og-dark-big.png" width="700"></a>

**Comment by seneca** [comment]
Wonderful news. Thank you for all your hard work. I'm looking forward to going over all the docs and checking all the new goodies you have provided.
Nuejs needs a major push in terms of exposing it a wider audience.

**Reply by nobkd** [reply]
https://github.com/nuejs/nue/discussions/333#discussioncomment-10771061

**Reply by tipiirai** [reply]
Thanks guys! 

I'm kind of good at marketing up to the point when I need to expose things to people. So quite terrible in fact. Just waiting for it to happen at some point.

**Comment by JensRoland** [comment]
Loved the article, and the facelift for the homepage as well!

I am a big fan of Nue and a fervent believer in what you are trying to do with it, so take the following in the spirit of friendship:

I was put off by the new branding of Nue as a "Design Engineering" framework.

I am not familiar with the term 'Design Engineering', but the wording plus the visual cues in the header gives an unmistakeable 'Figma dude' vibe...... which - if I didn't know better - could have ended my interest in Nue right then and there.

That scares me, because I want people like me to enter nuejs.org for the first time and _feel like they've come home after a long cold frontend winter_.

I don't identify as a Figma dude. I'm a software developer and engineering lead with 25 YoE with HTML, CSS and JS. I know my way around UX and CSS, but when it comes to design, I leave that to the professionals who know their way around InVision, Photoshop, and Figma.

The new frontpage to me screams "DO YOU LIKE FIGMA? THEN YOU'LL ♥ NUE!".

Is that who Nue is for? Most of the Figma dudes I have worked with are clueless about HTML and CSS and are constantly looking for the next AI tool to auto-translate their Figma designs to React so they can finally stop having to deal with developers and forget that web technologies exist.

I actually have the utmost respect for designers; designers play a crucial part in web development and their job is highly complex, but they don't typically know a lot about web technologies and I suspect they aren't the core audience for Nue because the main thing they want from HTML and CSS is to never see them again.

I am guessing there is a definition of 'design engineering' out there and that it is a lot less design and a lot more engineering than that. But if people aren't familiar with the term then they will look to the visual cues and subheader for clarification, and here I think the visual cues are betraying Nue a bit.

**Reply by tipiirai** [reply]
I hear you. I actually though the previous version was screaming Figma too much, and wanted to come closer to engineering and the basics of web. For visual cues, I guess you refer to the selector rectangle around the word Design? I'm happy to correct the tone in there, but still want to keep the word "Design Engineering" because I feel it's very close to what Nue aims to be.

**Reply by tipiirai** [reply]
Or maybe you're right. "Design Engineering" might not describe the product as well as say "The simple web framework" or "The simple web stack" or even "The standard web stack". Thoughts?

**Reply by JensRoland** [reply]
All of those sound better to me than the 'design engineering' one. If I had to pick one, maybe "The simple web stack"? It's six and two threes, really.

Coming up with a strong brand identity for a JS/web framework is hard, not least because there are >50 competing frameworks vying for the same users and each one promises roughly the same things: **fast, lightweight, productive, progressive, modern, web standards**, blah blah.

It is easy to get hype fatigue in this space, and how do you stand out without missing the mark? How do you tell users "_Nue is all those things the other ones claim, only for real this time._"?

Every framework wants to solve many of the same problems and if framework X is just a little better here or there then ...big deal? I'm not going to migrate a large codebase or learn a new framework for a 10% speedup or a slightly nicer reactive signaling syntax. What draws me to Nue is its dedication to things that matter to me:

- Obsessive focus on **productivity** and true **developer ergonomics**
- '**Just Works**' philosophy: maybe I am projecting here, but that's the sense I get
- **Lightweight**, _not just the JS_ but the HTML and the CSS too -- and designed to make _my own_ JS, HTML and CSS lightweight too.
- **Not reinventing the wheel**: Tailwind is just the `style` attribute with extra steps; leveraging existing **web standards** means Nue users are learning to use the Web, not pigeonholing themselves into a career as a 'React Andy'
- **Pragmatism**: the right balance between size and features, performance vs. operational simplicity, convention vs. configuration. Framework X may be 60% faster at sorting 100k table rows, but if the difference is measured in milliseconds and framework X achieved the speedup by cutting key features and mangling the DX then it's not for me
- Driven by **real world benefits** rather than oversold or niche gimmicks
- **Batteries included**: It's not just a JS framework, it's a full suite for building MPAs. I do dream of more serverful features/patterns (lightweight MVC stuff, API endpoints, database use cases) in Nue, but I can be patient

Unfortunately, turning all that into a pithy 7-word hero header is an artform that I never fully mastered.

**Reply by nonfungibletunji** [reply]
the design web stack

**Comment by k2fx** [comment]
Please compare to 11ty...
